src/PlaywrightSharp.Chromium/ChromiumBrowserApp.cs

```diff
@@ -10,10 +10,12 @@ namespace PlaywrightSharp.Chromium
     public class ChromiumBrowserApp : IBrowserApp

```

Comment:
FYI. My next task is making this class a generic class. There is not `crBrowserApp` un Playwright.